tools/osbuild-dev

```diff
@@ -93,20 +115,19 @@ class Manifest:
 
         # We can't handle all source types but some we can
         if "org.osbuild.curl" in self.data["sources"]:
-            for name, source in self.data["sources"]["org.osbuild.curl"][
+            for hasj, url in self.data["sources"]["org.osbuild.curl"][
```

Comment:
nitpick: `hasj`? is it a typo of `hash`? :thinking:

Reply:
No, `hash` is a built-in name that I don't want to shadow.